framework/set/setRKE2K3s.go

```diff
@@ -157,15 +135,14 @@ func SetRKE2K3s(t *testing.T, k8sVersion string, nodePools []config.Nodepool, fi
 
 		machineConfigBlockBody.SetAttributeRaw("name", name)
 
-		num++
+		count++
 	}
 
-	_, err = file.Write(f.Bytes())
-
+	_, err = file.Write(newFile.Bytes())
 	if err != nil {
-		t.Logf("Failed to write RKE2/K3S configurations to main.tf file. Error: %v", err)
+		logrus.Infof("Failed to write AKS configurations to main.tf file. Error: %v", err)
```

Comment:
same comment as above -- copy/paste fluke -- this should mention RKE2/K3s -- not AKS :)

Reply:
Resolved